sdk/src/main/java/io/customer/sdk/repository/preference/SharedPreferenceRepository.kt

```diff
@@ -41,6 +44,10 @@ internal class SharedPreferenceRepositoryImp(context: Context) : SharedPreferenc
                 siteId = getString(SITE_ID, null).orEmpty(),
                 apiKey = getString(API_KEY, null).orEmpty(),
                 region = getString(REGION, null).toRegion(),
+                client = Client.fromRawValue(
+                    source = getString(CLIENT_SOURCE, null) ?: "Unknown",
```

Comment:
In the defaults of the class `CustomerIOStoredValues`, we use
```kotlin
Client.Android(sdkVersion = Version.version)
```
 but here we set `Unknown` if the value doesn't exist. 
 
 should we probably have both defaults the same?